app/src/ui/diff/image-diffs/difference-blend.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+import { DiffImage } from './diff-image'
+import { ICommonImageDiffProperties } from './modified-image-diff'
+
+export class DifferenceBlend extends React.Component<
+  ICommonImageDiffProperties,
+  {}
+> {
+  public render() {
+    const style: React.CSSProperties = {
+      height: this.props.maxSize.height,
+      width: this.props.maxSize.width,
+    }
+
+    const maxSize: React.CSSProperties = {
+      maxHeight: this.props.maxSize.height,
+      maxWidth: this.props.maxSize.width,
+    }
+
+    return (
+      <div className="image-diff-difference" ref={this.props.onContainerRef}>
+        <div className="sizing-container" ref={this.props.onContainerRef}>
```

Comment:
This `onContainerRef` shouldn't be necessary if it's also attached to the outer div, right?